components/BottomNavigation/src/MDCBottomNavigationBar.h

```diff
@@ -123,6 +123,19 @@ typedef NS_ENUM(NSInteger, MDCBottomNavigationBarAlignment) {
  */
 @property(nullable, nonatomic,copy) UIColor *backgroundColor UI_APPEARANCE_SELECTOR;
 
+/**
+ The inset applied to each items bounds to determine the rect in which the items' contents will be
+ centered. The contents are centered in this rect, but not compressed, so they may still extend
+ beyond these bounds. Defaults to {0, 0, 0, 0}.
+ */
+@property(nonatomic, assign) UIEdgeInsets itemsContentInsets;
```

Comment:
Looks like these are flipped for RTL. If that is correct, please document the behavior.